generated-source-files.mk

```diff
@@ -32,36 +32,39 @@ genny-map-all: genny-map-client-received-blocks genny-map-storage-block-retrieve
 # Map generation rule for client/receivedBlocksMap
 .PHONY: genny-map-client-received-blocks
 genny-map-client-received-blocks: install-m3x-repo
-	cd $(m3x_package_path) && make hashmap-gen \
-		pkg=client \
-		key_type=idAndBlockStart \
-		value_type=receivedBlocks \
-		out_dir=$(m3db_package_path)/client \
-		rename_type_prefix=receivedBlocks
+	cd $(m3x_package_path) && make hashmap-gen  \
+		pkg=client                                \
+		key_type=idAndBlockStart                  \
+		value_type=receivedBlocks                 \
+		target_package=$(m3db_package)/client     \
+		rename_type_prefix=receivedBlocks         \
+	  temp_suffix=temp
```

Comment:
As discussed in m3x PR, kinda lame that we have to specify temp_suffix - everyone will want to use the same temp suffix assumedly?